lib/config/options/index.ts

```diff
@@ -223,6 +223,10 @@ const options: RenovateOptions[] = [
     allowedValues: ['disabled', 'enabled', 'reset'],
     stage: 'repository',
     default: 'disabled',
+    experimental: true,
+    experimentalDescription:
+      'This is an experimental feature and may be modified or removed in a future non-major release.',
+    experimentalIssues: [6589],
```

Comment:
Note that issue https://github.com/renovatebot/renovate/issues/6589 might not be what we want here.
should we open an issue for it or should i just remove it?